phy/phy.nim

```diff
@@ -198,22 +208,34 @@ proc compile(tree: var PackedTree[spec.NodeKind], source, target: Language) =
       assert false, "cannot be handled here: " & $current
     of lang1:
       syntaxCheck(tree, lang1_checks, module)
-      # TODO: don't hardcode the pointer size
-      tree = tree.apply(pass1.lower(tree, 8))
+      tree = tree.apply(pass_inlineTypes.lower(tree))
       current = lang0
+    of lang2:
+      syntaxCheck(tree, lang2_checks, module)
+      # TODO: don't hardcode the pointer size
```

Comment:
does it make sense to lift this out as a const for now, so it's easier to change to a variable that's a bit more dynamic taking configuration into account?

Reply:
Yep, good call.